### PR TITLE
#43 database seed data and smaller changes

### PR DESCRIPTION
**Pull request by RaidenRabit**
database script is run everytime a Rebuild action is performed at wcfService project level

### CODE REVIEW

File: Code/GeorgiaLibrarySystem-/GtlService/GtlService.csproj [hunk 1]
```diff
@@ -146,11 +146,23 @@
...
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
   -->
+  <Target Name="OnCleanOnly" AfterTargets="Clean">
+    <Exec Command="&#xD;&#xA;	echo ufo&#xD;&#xA;                   cd ..\..\Database&#xD;&#xA;	call ScriptDatabase.bat&#xD;&#xA;                   " />
+    <Message Text="UFO invasion detected" Importance="High" />
```
Comment (bubriks) [inline Code/GeorgiaLibrarySystem-/GtlService/GtlService.csproj:166]: what is this...

File: Code/Database/SQLCreateQuery.sql [hunk 2]
```diff
@@ -1,6 +1,8 @@
 USE master
-
-DROP DATABASE GTL;
+go
+IF DB_ID('GTL') IS NOT NULL
+ALTER DATABASE GTL SET SINGLE_USER WITH ROLLBACK IMMEDIATE
+DROP DATABASE IF EXISTS GTL;
```
Comment (bubriks) [inline Code/Database/SQLCreateQuery.sql:5]: what are the things being added here?

File: Code/GeorgiaLibrarySystem-/GtlWebsite/Web.config [hunk 3]
```diff
@@ -53,9 +53,7 @@
       </basicHttpBinding>
     </bindings>
     <client>
-      <endpoint address="http://localhost:52690/Controller/LoginService.svc"
-        binding="basicHttpBinding" bindingConfiguration="BasicHttpBinding_ILoginService"
-        contract="LoginServiceReference.ILoginService" name="BasicHttpBinding_ILoginService" />
+      <endpoint address="http://localhost:52690/Controller/LoginService.svc" binding="basicHttpBinding" bindingConfiguration="BasicHttpBinding_ILoginService" contract="LoginServiceReference.ILoginService" name="BasicHttpBinding_ILoginService" />
```
Comment (bubriks) [inline Code/GeorgiaLibrarySystem-/GtlWebsite/Web.config:56]: why was this changed?

Comment (RaidenRabit) [reply Code/GeorgiaLibrarySystem-/GtlWebsite/Web.config:56]: no idea why this changed

File: Code/Database/SQLCreateQuery.sql (hunk 2 above)
Comment (RaidenRabit) [reply Code/Database/SQLCreateQuery.sql:5]: to make sure that even if the db don't exist in the first place, sqlcmd can still run the script fully.
the Alter db part is because sometimes ssms or some other process randomly blocks sqlcmd access to eventually existing gtl db.

File: Code/GeorgiaLibrarySystem-/GtlService/GtlService.csproj (hunk 1 above)
Comment (RaidenRabit) [reply Code/GeorgiaLibrarySystem-/GtlService/GtlService.csproj:166]: pre-build action. it makes it run the sql script before going on forward with normal visual studio/c# stuff. also prints UFO invasion detected in the output console :D

**Review by bubriks — APPROVED** [review]
i guess ok, but idk how well itll work and didnt quite get how to use it (maybe post some description in group)
